manly.py

```diff
@@ -25,9 +25,18 @@
 import sys
 
 
-_ANSI_BOLD = "\033[1m%s\033[0m"
-if not sys.stdout.isatty():
-    _ANSI_BOLD = "%s"
+# A python3.5 backport to cover differences between 2/3.4 and 3.5
+class CalledProcessError(subprocess.CalledProcessError):
+    def __init__(self, returncode, cmd, output=None, stderr=None):
+        self.returncode = returncode
+        self.cmd = cmd
+        self.output = output
+        self.stderr = stderr
```

Comment:
If you think it makes sense, please add one or two more comments to this explaining what changed, or at least reference the changes and github issue #27